tests/queries/0_stateless/02891_rename_table_without_keyword.sql

```diff
@@ -0,0 +1,38 @@
+DROP DATABASE IF EXISTS rename_db;
```

Comment:
JFYI consider using `{CLICKHOUSE_DATABASE:Identifier}` query parameter to avoid adding `no-parallel` tag